pkg/operator/provisioner/provisioner.go

```diff
@@ -54,14 +60,15 @@ type provisionerConfig struct {
 	clusterName string
 }
 
-func newRookVolumeProvisioner(clusterManager *clusterManager) controller.Provisioner {
-	return &rookVolumeProvisioner{
-		clusterManager: clusterManager,
+// NewRookVolumeProvisioner creates RookVolumeProvisioner
+func NewRookVolumeProvisioner(client kubernetes.Interface) controller.Provisioner {
```

Comment:
`Provisioner` seems fine since you know what package you're in, or maybe `VolumeProvisioner`. Rook is obvious, right?